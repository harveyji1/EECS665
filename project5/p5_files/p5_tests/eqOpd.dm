student: class{
    studentID: int;
};

fn: () int{
    hello: int = 1;
    myStudent: student;
    if(myStudent==1){
        give hello;
    }
}

main: () void{
    fn();
}