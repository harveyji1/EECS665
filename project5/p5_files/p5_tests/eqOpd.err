FATAL [9,8]-[9,18]: Invalid equality operation
Type Analysis Failed
