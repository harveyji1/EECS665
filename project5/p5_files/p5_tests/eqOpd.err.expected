FATAL [8,8]-[8,17]: Invalid equality operand
Type Analysis Failed